### PR TITLE
Word of the day #3

### PR DESCRIPTION
**Pull request by JKtranslator**
Block Time
Bridge
Genesis Block
Hard Fork
UTXO

### CODE REVIEW

File: docs/project/glossary.md [hunk 1]
```diff
@@ -25,14 +25,27 @@ It is paid in the blockchain’s native token. It is usually higher when the net
...
 
 It can be measured in different ways. In some blockchains, it is expressed in how much actual data the block can carry (for example, in Zcash, the block size is 2MB). In other blockchains, the block size is related to the computational processing limit it can consume from the network (usually expressed in gas). Ethereum’s and Alephium’s block sizes are measured this way.
 
+### Block Time
 
+Block Time is the time needed to compute the transactions inside a block and send it to the network.
 
+Transactions are gathered inside a block and checked by the miners (or validators on PoS blockchains). Usually, the Block Time is impacted by the mining difficulty, as it is adjusted to reflect the network's computational capacity (hashrate) over a given time.
+
+Alephium network has a difficult adjustment on every block and has an expected block time of **64 seconds**.
```
Comment (Sakrecoer) [inline docs/project/glossary.md]: alephium has a _difficulty_ adjustement on....

File: docs/project/glossary.md [hunk 2]
```diff
@@ -25,14 +25,27 @@ It is paid in the blockchain’s native token. It is usually higher when the net
...
+
+Adittional resources: [Block Time and Block Size Article](https://medium.com/@alephium/block-time-and-block-size-16e37292444f)
```
Comment (Sakrecoer) [inline docs/project/glossary.md]: Additional (two "D" one "T")

File: docs/project/glossary.md [hunk 3]
```diff
@@ -58,8 +71,22 @@ This is the monetary value of the gas. Gas is defined as the computational effor
 
 At Alephium, the smallest price denomination is 0.0000001 ALPH, also called 1 phi.
 
+### Genesis Block
+
+A Genesis Block is the name of a blockchain’s first block ever mined. As the blocks get layered—one on top of the other, the Genesis Block is the foundation or beginning of it.
```
Comment (Sakrecoer) [inline docs/project/glossary.md]: "As the blocks get layered on top of each other." 

Maybe it's enough to either remove the `—`

File: docs/project/glossary.md [hunk 4]
```diff
@@ -148,6 +172,14 @@ Additional resources: [Transaction fee GitHub Implementation](https://github.com
 
 ## U
 
+### UTXO
+
+[UTXO](https://en.wikipedia.org/wiki/Unspent_transaction_output) (unspent transaction output) is the term for the amount of a specific currency that remains unspent after a cryptocurrency transaction.
```
Comment (Sakrecoer) [inline docs/project/glossary.md]: (Unspent Transaction Output) 
I would capitalize the first letter to emphasis that it is the long version of the acronym.

**Review by Sakrecoer — CHANGES_REQUESTED** [review]
Thanks @JKtranslator, found a couple of typos and suggestions. Please adjust accordingly. I'll gladly merge tomorrow morning if you'd rather call today a day :)

File: docs/project/glossary.md (hunk 1 above)
Comment (JKtranslator) [reply docs/project/glossary.md]: changed

File: docs/project/glossary.md (hunk 2 above)
Comment (JKtranslator) [reply docs/project/glossary.md]: changed

File: docs/project/glossary.md (hunk 3 above)
Comment (JKtranslator) [reply docs/project/glossary.md]: changed

File: docs/project/glossary.md (hunk 4 above)
Comment (JKtranslator) [reply docs/project/glossary.md]: changed

**Review by Sakrecoer — APPROVED** [review]
Awesome! Good catch on ttitle capitalization!! Many thanks! Have a great day!!

**Pull request merged**